viewing_party/party.py

```diff
@@ -1,23 +1,127 @@
 # ------------- WAVE 1 --------------------
 
+from typing import Counter
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if bool(title) == True and bool(genre) == True and bool(rating) == True:
+        return new_movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+    return user_data
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
+def get_watched_avg_rating(user_data):
+    
+    ratings = [movie["rating"] for movie in user_data["watched"]]
+    Sum = sum(ratings)
+    Count = len(ratings)

```

Comment:
Variables in Python should begin with a lower case letter.  I understand you not wanting to use `sum` again though because it's a built in function:

```suggestion
    rating_sum = sum(ratings)
    rating_count = len(ratings)
```